pkg/logger/logger.go

```diff
@@ -24,6 +26,8 @@ var (
 
 func init() {
 	flag.IntVar(&logLevel, "log-level", 0, "Use zap-core log system.")
+	logf.SetLogger(ZapLogger())
+	Log = logf.Log.WithName("nsx-operator")
```

Comment:
logf.Log.WithName("nsx-operator-logger")? is better?

Reply:
Just nsx-operator, it's enough.